public/js/app/counter.js

```diff
@@ -0,0 +1,26 @@
+const delay = 3000;
+
+$("#game_counter").parent().hide().fadeIn();
+            
+setInterval(function(){
+    $.get('lobby_api', { resource: "players"}, function (body) {
```

Comment:
```
setInterval(function() {
    $.get('lobby_api', {resource: "players"}, function (body) {
```

Well, you're mixing `() => {}` and `function () {}`. Choose one of this.

Reply:
> Well, i think, i think... you can create cache system, ask player count on server side, store result to cache, and read it from cache to outside world.
> 
> 1. less queries to db
> 2. you'll have data already from cache and don't have to show 0 after opening the page.

We don't want caching for this, because we want the player count to be LIVE on the webpage!
Unless we fake it by adding random deviations to a cached number, but this will be easy to tell apart...

Also, no database is involved in that query. It's using the python server API.